tests/browser_test.py

```diff
@@ -37,12 +38,10 @@ async def test_review_signoff(
         permissions={"write": [editor_id, reviewer_id]},
         if_not_exists=True,
     )
-    await client.patch_group(
-        id="product-integrity-editors", data={"members": [editor_id]}
-    )
-    await client.patch_group(
-        id="product-integrity-reviewers", data={"members": [reviewer_id]}
-    )
+    data = JSONPatch([{"op": "add", "path": "/data/members/0", "value": editor_id}])
+    await client.patch_group(id="product-integrity-editors", changes=data)
+    data = JSONPatch([{"op": "add", "path": "/data/members/0", "value": reviewer_id}])
+    await client.patch_group(id="product-integrity-reviewers", changes=data)
```

Comment:
In response to https://github.com/mozilla-services/kinto-dist/issues/1965#issuecomment-1008960512